data/mods/Magiclysm/monsters/forgedwellers.json

```diff
@@ -0,0 +1,145 @@
+    "copy-from": "mon_lizardfolk_shaman",
+    "default_faction": "wonder_forge",
+    "aggression": 0,
+    "special_attacks": [ { "type": "spell", "spell_data": { "id": "laze" }, "cooldown": 10 }, [ "scratch", 5 ] ],
+    "death_drops": {
+      "subtype": "collection",
+      "groups": [ [ "subway", 40 ], [ "sewer", 20 ], [ "trash", 5 ], [ "bedroom", 1 ], [ "dresser", 5 ], [ "ammo", 18 ] ]
+    }
+  },
+  {
+    "id": "mon_forgemaster3",
+    "type": "MONSTER",
+    "name": { "str": "gracken forgemaster" },
+    "description": "This is some form of eldritch monstrosity; an uncouth black being with smooth, oily, skin and unpleasant horns that curve inward toward each other.  Tall and thin, the shadows cling unnaturally to its vaguely defined humanoid form as it shuffles along, its hands twitching and spasming so rapidly as to appear a little more than a black blur of claws.  This one politely asks you to mind your step as this is a working environment.",
+    "default_faction": "wonder_forge",
+    "copy-from": "mon_gracke",
+    "death_drops": {
+      "subtype": "collection",
+      "groups": [ [ "subway", 40 ], [ "sewer", 20 ], [ "trash", 5 ], [ "bedroom", 1 ], [ "dresser", 5 ], [ "ammo", 18 ] ]
+    }
+  },
+  {
+    "id": "mon_forgedemons",
+    "type": "MONSTER",
+    "name": { "str": "forge demon" },
+    "description": "The first thing you notice is that this creature has five arms.  The next is that it walks on three legs. It walks amongst the vault until it notices you then throws itself at the walls screaming bloody murder.  Its mere presence fills you with an unfathomable dread.",
+    "default_faction": "wonder_forge",
+    "bodytype": "human",
+    "species": [ "NETHER" ],
+    "diff": 5,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 400,
+    "speed": 80,
+    "material": [ "flesh" ],
+    "symbol": "&",
+    "color": "white",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 7,
+    "melee_dice": 5,
+    "melee_dice_sides": 6,
+    "melee_cut": 6,
+    "dodge": 4,
+    "vision_day": 30,
+    "harvest": "demihuman_large_fur",
+    "special_attacks": [ { "type": "spell", "spell_data": { "id": "demon_breath_fire", "min_level": 7 }, "cooldown": 10 }, [ "scratch", 15 ] ],
+    "death_function": [ "NORMAL" ],
+    "flags": [
+      "SEES",
+      "SMELLS",
+      "HEARS",
+      "WARM",
+      "BASHES",
+      "ANIMAL",
+      "PATH_AVOID_DANGER_1",
+      "NO_BREATHE"
+    ]
+  },
+  {
+    "id": "mon_forgedwarves",
+    "type": "MONSTER",
+    "name": { "str": "forge workers" },
+    "description": "A squat humanoid that looks quite sturdily built.  They style their hair in mohawks and other strange fashions that seem to be dictated by what parts of their skin aren't scarred from burns.  Their clothes are designed to protect them from the intense environment they work in.",
```

Comment:
If they're wearing protective equipment for the forge, you could add some armor_fire values. Applies to the rest of the forge faction.